virtcontainers/kata_agent.go

```diff
@@ -532,25 +532,28 @@ func (k *kataAgent) updateInterfaces(interfaces []*vcTypes.Interface) error {
 }
 
 func (k *kataAgent) updateRoutes(routes []*vcTypes.Route) ([]*vcTypes.Route, error) {
-	if routes != nil {
-		routesReq := &grpc.UpdateRoutesRequest{
-			Routes: &grpc.Routes{
-				Routes: k.convertToKataAgentRoutes(routes),
-			},
-		}
-		resultingRoutes, err := k.sendReq(routesReq)
-		if err != nil {
-			k.Logger().WithFields(logrus.Fields{
-				"routes-requested": fmt.Sprintf("%+v", routes),
-				"resulting-routes": fmt.Sprintf("%+v", resultingRoutes),
-			}).WithError(err).Error("update routes request failed")
-		}
-		resultRoutes, ok := resultingRoutes.(*grpc.Routes)
-		if ok && resultRoutes != nil {
-			return k.convertToRoutes(resultRoutes.Routes), err
+	// XXX Remove me
```

Comment:
just commenting to remind us in future that the commented code needs to be un-commented.

Reply:
This will go away when we add metric to the route